_posts/2019-06-16-good-first-issue.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "How I built Good First Issue bot with OpenFaaS Cloud"
+description: In this post Rajat will share what triggered him to write goodfirstissue bot, where was it deployed and how it was a nobrainer to move to OpenFaaS cloud.
```

Comment:
nobrainer -> no-brainer

Perhaps there is a more international-friendly term? https://www.thesaurus.com/browse/no-brainer

Reply:
changed it to "how it was an easy decision to move..."